### PR TITLE
[BACKLOG-2452] - Added ability to not show private endpoints. By default...

### PR DESCRIPTION
**Pull request by diogofscmariano**
... they are not visible. Fixed some hardcoded paddings

@pamval please review

### CODE REVIEW

File: src/org/pentaho/di/baserver/utils/inspector/Endpoint.java [hunk 1]
```diff
@@ -63,12 +65,28 @@ public void setHttpMethod( HttpMethod httpMethod ) {
     return this.queryParams;
   }
 
-  public String getDoc() {
-    return doc;
+  public boolean isDeprecatted() {
```
Comment (pamval) [inline src/org/pentaho/di/baserver/utils/inspector/Endpoint.java]: typo

File: src/org/pentaho/di/baserver/utils/inspector/Endpoint.java [hunk 2]
```diff
@@ -63,12 +65,28 @@ public void setHttpMethod( HttpMethod httpMethod ) {
...
+    return deprecated;
+  }
+
+  public void setDeprecated( boolean deprecatted ) {
```
Comment (pamval) [inline src/org/pentaho/di/baserver/utils/inspector/Endpoint.java]: typo

**Comment by pamval** [comment]
Updates for the unit tests ?

**Comment by diogofscmariano** [comment]
PR updated. @pamval please review

**Pull request merged**